docs/Project.toml

```diff
@@ -1,6 +1,3 @@
 [deps]
 AbstractAlgebra = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
 Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
-
-[compat]
-Documenter = "^0.23.0"
```

Comment:
Of course we can also simply increase this to 0.26, if you prefer to pin the version. Let me know what you prefer

Reply:
We used to pin the exact version, since at some point there were breaking changes when they went from v0.10 to v0.11 (the numbers are not right, but you get the idea). They actually advice to pin the minor version to avoid the documentation from silently breaking.

Reply:
Of course part of the whole idea of testing this in a CI system is to make sure the documentation is *not* breaking silently, but rather is breaking loudly :-). The drawback of pinning a version is that you don't get to benefit from fixes and improvements, and easily an forget to update it. The advantage indeed is that you don't have to fight with changes that break you stuff (but that's kinda going to happen anyway when we change the version of Julia used to build the manual / run doctests. Ah well).

Anyway, I have pinned it now!

Unfortunately, I don't think the current test will actually report doctest breakage; I am experimenting with that now.

Reply:
I personally prefer the reproducibility. Thanks for pinning it.